### DISCUSSION TITLE
Tina-Cloud-Starter Landing Page

### DISCUSSION THREAD

**Discussion by 0xShakhawat** [tinacms/tinacms#2575] [Help] [answered]
From Tina-Cloud-Starter, How can I set blog page as my homepage?
I love Minimal Blog Themes, How can I find minimal themes that suitable for Tina.io

_sorry for my English_

**Comment by perkinsjr** [comment, marked as answer]
Hi @shakhawat-pi thanks for reaching out about setting the blog page as your homepage. 

You have a few options to achieve this: 

1. You can change the redirect in the next.config.js file from 

```javascript
module.exports = withSvgr({
  async rewrites() {
    return [
      {
        source: "/",
        destination: "/home",
      },
    ];
  },
});
```
to 
```javascript
module.exports = withSvgr({
  async rewrites() {
    return [
      {
        source: "/",
        destination: "/posts",
      },
    ];
  },
});
```
2. You can remove all the pages you wouldn't need create an `index.tsx` file in the pages directory that would hold all the posts from the `posts.tsx` and remove the rewrite mentioned above. This would be a more permanent solution if you planned to never use any other pages again.

Finally in regards to your question about "how can you find minimal themes that are sutitable for tina" , you can use any blog that is next.js and uses markdown or mdx files. For example:

[Next.js blog starter](https://github.com/vercel/next.js/tree/canary/examples/blog-starter)
[Next.js Tailwind Starter blog](https://jamstackthemes.dev/theme/nextjs-tailwind-starter-blog/)

Hope this helps!
